docs/testrunner-toolkit.md

```diff
@@ -82,97 +62,82 @@ Each docker image tag is the 'latest' image that supports the specific framework
 |TestCafe Version|Supported Platforms|Supported Browsers|
 |-----|-----|-----|
 |1.11.0|macOS 11.00|Safari, Chrome, Firefox, MicrosoftEdge|
+|1.11.0|Windows 10|Chrome, Firefox, MicrosoftEdge|
 |1.10.1|Windows 10|Chrome, Firefox, MicrosoftEdge|
 
 </TabItem>
 </Tabs>
 
-<!--END_AUTO_GENERATED_TABLE-->
 
 ### Supported Frameworks in Docker Runner
 
+Browser support or each framework is based on the Sauce Labs docker images provided in the `saucectl` installation. Each Docker image tag is the latest image that supports the specific framework version, as detailed in the available release notes.
+
 <Tabs
   defaultValue="cypress"
   values={[
     {label: 'Cypress', value: 'cypress'},
     {label: 'Playwright', value: 'playwright'},
-    {label: 'Puppeteer', value: 'puppeteer'},
```

Comment:
We do support puppeteer --> https://github.com/saucelabs/saucectl-puppeteer-example/

Reply:
I thought it wasn't official yet, since it was commented out in the sauce-cloud section. I will put it back for Docker. Thanks.